utils.py

```diff
@@ -1,6 +1,6 @@
 from aiohttp.web import Request, HTTPNotImplemented, HTTPForbidden, HTTPUnauthorized, FileField
 
-import config
+from config import global_config
```

Comment:
I know that's how it already was, but please move this below the built-in imports (with a newline between)